diff --git a/node_modules/@types/responselike/index.d.ts b/node_modules/@types/responselike/index.d.ts
index 7152388..5abd367 100644
--- a/node_modules/@types/responselike/index.d.ts
+++ b/node_modules/@types/responselike/index.d.ts
@@ -8,7 +8,6 @@
 import { IncomingMessage } from 'http';
 import { Stream } from 'stream';
 
-export = ResponseLike;
 
 /**
  * Returns a streamable response object similar to a [Node.js HTTP response stream](https://nodejs.org/api/http.html#http_class_http_incomingmessage).
@@ -32,3 +31,5 @@ declare class ResponseLike extends Stream.Readable {
         url: string
     );
 }
+
+export default ResponseLike;
